subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -56,14 +58,14 @@ fun randomFloat(): Float {
 }
 
 /**
- * Delegate of [Random.nextLong] ()}
+ * @return value between 1 and Long.MAX_VALUE
  */
 fun randomLong(): Long {
-    return random.nextLong()
+    return 1L + random.nextLong(Long.MAX_VALUE - 1L)
```

Comment:
```suggestion
    return random.nextLong(Long.MAX_VALUE - 1L)
```
Zero is a valid value for dates.

Reply:
I made it like that because of `randomInt` code. Maybe I need to change it too?

Reply:
Yes, see no reasons to start from 1.

Reply:
Done